packages/edit-site/src/components/add-new-template/new-template-part.js

```diff
@@ -35,11 +35,17 @@ export default function NewTemplatePart( { postType } ) {
 		}
 
 		try {
+			// Currently template parts only allow latin chars.
+			// Fallback slug will receive suffix by default.
+			const cleanSlug =
+				kebabCase( title ).replace( /[^\w-]+/g, '' ) ||
+				'wp-custom-part';
```

Comment:
Could we extract this into a utility function or hook?

Reply:
We have the [`cleanForSlug`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-url/#cleanforslug) method, which supports non-Latin chars; this is the reason I decided to inline it to avoid confusion in the future.

Reply:
We could still name this `getCleanSlugForLatinChars` or something to make it explicit, but I don't feel strongly that we need to do this 👍 .

Reply:
I will leave inline for now, if that's okay for you.